pkg/workflows/wasm/pb/wasm.proto

```diff
@@ -66,6 +66,10 @@ message FetchRequest {
   bytes body = 4;
   uint32 timeoutMs = 5;
   string id = 6;
+  string workflowId = 7;
```

Comment:
@vyzaldysanchez Could we add a comment here to explain why we're adding this here? (It's because it's needed for the ID of the request + for metrics

Reply:
On it.

Reply:
Actually it's probably worth pulling this out into a metadata sub-message to not pollute the overall message